manuscript/references.bib

```diff
@@ -310,3 +310,154 @@ @article{pomdp-intro
+}
+
+@article{A3C,
+	title = {Reinforcement {Learning} through {Asynchronous} {Advantage} {Actor}-{Critic} on a {GPU}},
+	url = {http://arxiv.org/abs/1611.06256},
+	urldate = {2021-05-04},
+	journal = {arXiv:1611.06256 [cs]},
+	author = {Babaeizadeh, Mohammad and Frosio, Iuri and Tyree, Stephen and Clemons, Jason and Kautz, Jan},
+	month = mar,
+	year = {2017},
+	note = {arXiv: 1611.06256},
+	keywords = {Computer Science - Machine Learning},
+	file = {arXiv Fulltext PDF:/Users/marcus/Zotero/storage/9LSTHWSZ/Babaeizadeh et al. - 2017 - Reinforcement Learning through Asynchronous Advant.pdf:application/pdf;arXiv.org Snapshot:/Users/marcus/Zotero/storage/LLSZ73U2/1611.html:text/html},
+}
+
+@article{TRPO,
+	title = {Trust {Region} {Policy} {Optimization}},
+	url = {http://arxiv.org/abs/1502.05477},
+	urldate = {2021-05-04},
+	journal = {arXiv:1502.05477 [cs]},
+	author = {Schulman, John and Levine, Sergey and Moritz, Philipp and Jordan, Michael I. and Abbeel, Pieter},
+	month = apr,
+	year = {2017},
+	note = {arXiv: 1502.05477},
+	keywords = {Computer Science - Machine Learning},
+	annote = {Comment: 16 pages, ICML 2015},
+	file = {arXiv Fulltext PDF:/Users/marcus/Zotero/storage/DIWQTEK3/Schulman et al. - 2017 - Trust Region Policy Optimization.pdf:application/pdf;arXiv.org Snapshot:/Users/marcus/Zotero/storage/BFEAQCI7/1502.html:text/html},
+}
+
+@article{APE-X,
+	title = {Distributed {Prioritized} {Experience} {Replay}},
+	url = {http://arxiv.org/abs/1803.00933},
+	urldate = {2021-05-04},
+	journal = {arXiv:1803.00933 [cs]},
+	author = {Horgan, Dan and Quan, John and Budden, David and Barth-Maron, Gabriel and Hessel, Matteo and van Hasselt, Hado and Silver, David},
+	month = mar,
+	year = {2018},
+	note = {arXiv: 1803.00933},
+	keywords = {Computer Science - Machine Learning},
+	annote = {Comment: Accepted to International Conference on Learning Representations 2018},
+	file = {arXiv Fulltext PDF:/Users/marcus/Zotero/storage/JFVEXT7R/Horgan et al. - 2018 - Distributed Prioritized Experience Replay.pdf:application/pdf;arXiv.org Snapshot:/Users/marcus/Zotero/storage/EG8KS6IZ/1803.html:text/html},
+}
+
+@article{A2C,
+	title = {Asynchronous {Methods} for {Deep} {Reinforcement} {Learning}},
+	url = {http://arxiv.org/abs/1602.01783},
+	urldate = {2021-05-04},
+	journal = {arXiv:1602.01783 [cs]},
+	author = {Mnih, Volodymyr and Badia, Adrià Puigdomènech and Mirza, Mehdi and Graves, Alex and Lillicrap, Timothy P. and Harley, Tim and Silver, David and Kavukcuoglu, Koray},
+	month = jun,
+	year = {2016},
+	note = {arXiv: 1602.01783},
+	keywords = {Computer Science - Machine Learning},
+	file = {arXiv Fulltext PDF:/Users/marcus/Zotero/storage/SHHNFQPN/Mnih et al. - 2016 - Asynchronous Methods for Deep Reinforcement Learni.pdf:application/pdf;arXiv.org Snapshot:/Users/marcus/Zotero/storage/2HZ9JWWI/1602.html:text/html},
+}
+
+@article{DQN,
```

Comment:
I think we should (also) cite the version that appeared in _Nature_ as the canonical reference here.  (also I think I might have both already in the `.bib` file somewhere but doesn't really matter)